tests/integration_tests/shmap_collective_matmul_test.py

```diff
@@ -19,9 +19,9 @@
 
 import pytest
 
-from MaxText.globals import PKG_DIR
+from MaxText.globals import REPO_ROOT
 
-sys.path.append(os.path.join(os.path.dirname(PKG_DIR), "pedagogical_examples"))
+sys.path.append(os.path.join(REPO_ROOT, "pedagogical_examples"))
```

Comment:
No action required, outside the scope of this PR, but: I'm pretty sure pytest's testconf.py or pytest.ini can do sys.path setup, which is a bit more idiomatic and require less special setup stuff like computing REPO_ROOT.

Reply:
Yeah I mean I don't want any explicit `sys.path` manipulation in the codebase. Just trying to get the src layout transfer over the line first.